networking/ingress-controller-dnsmgt.adoc

```diff
@@ -0,0 +1,37 @@
+:_content-type: ASSEMBLY
+[id="ingress-controller-dnsmgt"]
+= Configuring an Ingress Controller for disabling DNS Management
+include::_attributes/common-attributes.adoc[]
+:context: ingress-controller-dnsmgt
+
+toc::[]
+
+As a cluster administrator, when you create an Ingress Controller, the Operator manages the DNS records by default. This has some limitations when the required DNS zone is different from the cluster DNS zone or when the DNS zone is hosted outside the cloud provider. In such scenarios, the Operator reports a degraded state after attempting to provision the required resources in the cloud provider because the wildcard DNS record can not be created on the cloud provider.
+
+As a cluster administrator, you can create or modify an Ingress Controller with `dnsManagementPolicy` to specify if the Operator needs to manage the DNS record lifecycle.
+
+== `Managed` DNS management policy
+The `Managed` DNS management policy for Ingress Controllers ensures that the lifecycle of the wildcard DNS record on the cloud provider is completely managed by the Operator.
+During transitioning an Ingress Controller from `Unmanaged` to `Managed` DNS management policy, the Operator attempts to create the DNS record on the cloud provider if it does not exist or updates the DNS record if it already exists.
+
+== `Unmanaged` DNS management policy
+The `Unmanaged` DNS management policy for Ingress Controllers ensures that the lifecycle of the wildcard DNS record on the cloud provider is not managed by the Operator, instead it is the responsibility of the cluster administrator.
+During transitioning an Ingress Controller from `Managed` to `Unmanaged` DNS management policy, the Operator does not clean up the previous wildcard DNS record provisioned on the cloud.
+
+Note that on the AWS cloud platform, if the domain on the Ingress Controller does not match with `dnsConfig.Spec.BaseDomain` then the DNS management policy is automatically set to `Unmanaged`.
+
+[NOTE]
+====
+Updating `dnsManagementPolicy` is not supported on default Ingress Controllers. As a cluster administrator, you have the discretion to create or modify the default Ingress Controller.
+====
```

Comment:
We can remove this note, we do support patching default ingress controllers as well.